### PR TITLE
Replace Zend_Translate with Symfony translator

### PR DESCRIPTION
**Pull request by jorissteyn**
This commit contains two important changes to make the translations
compatible with the Symfony translator:

 - translation files are renamed from languages/[lang].php to
   languages/messages.[lang].php

 - message placeholders are now using symfony-style named
   parameters ('%name%') instead of sprintf placeholders (%s, %d,
   %1$s)

EngineBlock users with a custom nl.php / en.php use instructions in
UPGRADING.md to easily convert the parameter style from the old to the
new format.

### CODE REVIEW

**Comment by tvdijen** [comment]
Maybe not for now, but would it be feasbile to use a placeholder for all occurences of 'institution' and make that one a configuration-setting?
95% of my changes to the language-files are now `s/institution/organization`.. 'Institition' is very typical for the educational world...

**Comment by thijskh** [comment]
And then also one for the suite name ("OpenConext") and product name "{{suite name Engineblock}}"

File: UPGRADING.md [hunk 1]
```diff
@@ -55,6 +55,21 @@ https://www.pivotaltracker.com/story/show/155358923
 
 Also see the upgraded [theme wiki][eb-wiki-theme-development] page.
 
+### Changes to translation files
+
+The following files have been renamed:
+
+ - lanauges/nl.php -> languages/messages.nl.php
```
Comment (MKodde) [inline UPGRADING.md]: Typo: ~lanauges~ languages

**Comment by jorissteyn** [comment]
Tim, Thijs, I'm working on a new PR with some finetuning and cleanup. Also removing some SURF-isms there and will take your comments into account.

**Comment by tvdijen** [comment]
You've just made my day!

File: UPGRADING.md (hunk 1 above)
Comment (MKodde) [reply UPGRADING.md]: I've taken the liberty to fix this typo by myself.

**Pull request merged**